ZenWithTerms/tla/ZenWithTerms.tla

```diff
@@ -29,6 +29,7 @@ VARIABLE descendant
 \* Values to bootstrap the cluster
 VARIABLE initialConfiguration
 VARIABLE initialValue
+VARIABLE initialCommittedVersion
```

Comment:
I would prefer a map here called `initialVersions` that maps each node to a version. We could then do
```
/\ initialVersions \in [Nodes -> InitialVersions]
/\ lastAcceptedVersion = initialVersions
```

and then further down below, just before the assertion, define

```
InitialCommittedVersions ==
  { version \in Range(initialVersions)
    : \E votes \in SUBSET(initialConfiguration)
    : /\ IsQuorum(votes, initialConfiguration)
      /\ \A n \in votes : version >= initialVersions[n] }
```

and then

```
CommittedValuesDescendantsFromInitialValue ==
  \E v \in InitialCommittedVersions :
    \A m \in messages :
      CommittedPublishRequest(m)
        =>
      [prevT |-> 0, prevV |-> v, nextT |-> m.term, nextV |-> m.version] \in descendant
```

I think this defines a stronger property (see `InitialCommittedVersions` and `CommittedValuesDescendantsFromInitialValue`), in particular stating that it's one of the actual initial versions that is being used.